enderio-decoration/src/main/java/com/enderio/decoration/common/network/ServerToClientLightUpdate.java

```diff
@@ -0,0 +1,37 @@
+package com.enderio.decoration.common.network;
+
+import java.util.function.Supplier;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraftforge.network.NetworkEvent.Context;
+
+public class ServerToClientLightUpdate {
+    public final BlockPos pos;
+    public final BlockState state;
+    
+    public ServerToClientLightUpdate(BlockPos pos, BlockState state) {
+        this.pos = pos;
+        this.state = state;
+    }
+    
+    public ServerToClientLightUpdate(FriendlyByteBuf buf) {
+        this.pos = buf.readBlockPos();
+        this.state = Block.stateById(buf.readVarInt());
+    }
+    
+    public void write(FriendlyByteBuf buffer) {
+        buffer.writeBlockPos(this.pos);
+        buffer.writeVarInt(Block.getId(this.state));
+    }
+    
+    static void handle(final ServerToClientLightUpdate message, Supplier<Context> ctx) {
+        ctx.get().enqueueWork(() ->
```

Comment:
Could this all be on one line?

Reply:
I will change it, however, this packet stuff will need to be merged with the one in base, so I might rewrite it a little to match that. I just needed a quick and dirty channel for this.

Reply:
Thats no problem at all, I'll handle that once this is merged :)